Modules/IO/MINC/include/itkMINCImageIO.h

```diff
@@ -55,6 +55,9 @@ struct ITKIOMINC_HIDDEN MINCImageIOPImpl;
  * reads the regularly sampled dimensions as I am not sure how to deal
  * with "iregularly sampled" dimensions yet!
  *
+ * Compression is supported with only the default compressor. The
+ * compression level option is supported in the range 0-9.
```

Comment:
Also right here, you are not exposing (implementation level) that max level is actually '9'. A program would have a hard time exposing this max value (slide bar) to the user if need be (and instead would display '100').

Reply:
The `ImageIOBase` class has a `GetMaximumCompressionLevel` method which would give the maximum value for a slider.